README.md

```diff
@@ -1,17 +1,20 @@
 # Homebrew webpage
 
-The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for the index is at layouts/index.html.
+The webpage uses [jekyll](https://github.com/mojombo/jekyll). The template for
+the index is at `_layouts/index.html`.
```

Comment:
Generally I don't see the advantage in hard-breaking lines in Markdown but don't mind doing it here.